hildon/hildon-gtk.c

```diff
@@ -31,6 +31,24 @@
 #include "hildon-edit-toolbar-private.h"
 #include "hildon-private.h"
 
+/* Add placeholder implementations for Maemo-specific Gtk functions */
```

Comment:
This comment seems excessive. It's to obvious what is happening here just looking at the code.
Adding comments for obvious things is not a good idea. But if you really think it's needed here, ok.
Just a recommendation.

Reply:
Moved to a different file